docker/config_template.yaml

```diff
@@ -86,6 +86,10 @@ services:
             statsd:
                 hostPort: {{ .Env.STATSD_ENDPOINT }}
                 prefix: "cadence-frontend"
+        {{- else if .Env.PROMETHEUS_ENDPOINT }}
+        metrics:
+            prometheus:
```

Comment:
prefer to add the timerType here, how about this
```yaml
timerType: {{ default .Env.PROMETHEUS_TIMER_TYPE "histogram" }}
```
then if "histogram" does not work for your setup, env PROMETHEUS_TIMERTYPE can be overrided to "summary"
ref: https://github.com/uber-go/tally/blob/1018d35f8be346a864b3ce25f2693c80da7c3fd6/prometheus/reporter.go#L254

Reply:
@wxing1292 good point, will do.